### PR TITLE
(DOCSP-34573): Add docs release notes for w/e November 24

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-34573

### CODE REVIEW

File: docs-release-notes.md [hunk 1]
```diff
@@ -1,37 +1,45 @@
+# November 24, 2023
+
+## Internal
+
+- Update Readability workflow to use upstream `docdoctor` repo with new build syntax
```
Comment (lindseymoore) [inline docs-release-notes.md:5]:
```suggestion
- Update Readability workflow to use upstream `docdoctor` repo with new build syntax.
```

File: docs-release-notes.md [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
+- Add `meta::description` to relevant pages to improve SEO for top 250 docs pages
```
Comment (lindseymoore) [inline docs-release-notes.md:6]:
```suggestion
- Add `meta::description` to relevant pages to improve SEO for top 250 docs pages.
```

File: docs-release-notes.md [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
+- Remove the `genre` tag from landing pages to improve docs search results
```
Comment (lindseymoore) [inline docs-release-notes.md:7]:
```suggestion
- Remove the `genre` tag from landing pages to improve docs search results.
```

File: docs-release-notes.md [hunk 4]
```diff
@@ -1,37 +1,45 @@
...
 - Landing Page: Add section for a selection of the SDK's example projects from the Example Projects page.
-- Realm Database/Model Data/Manage Realm Files/Encrypt a Realm: Add note that you must encrypt a realm on first open, or use ``writeCopy`` to create an encrypted realm. 
+- Realm Database/Model Data/Manage Realm Files/Encrypt a Realm: Add note that you must encrypt a realm on first open, or use `writeCopy` to create an encrypted realm.
 
 ## Kotlin SDK
 
-- Realm/Manage Realm Files/Encrypt a Realm: Add information on encryption for local and synced realms. 
+- Realm/Manage Realm Files/Encrypt a Realm: Add information on encryption for local and synced realms.
 - Connect to Atlas/Connect to App Services: Add information on encryption for app metadata.
-- Internal: Rename Kotlin Multiplatform Mobile (KMM) to Kotlin Multiplatform (KMP) in Kotlin template app ``README.md``, as KMM is deprecated.
```
Comment (lindseymoore) [inline docs-release-notes.md:12]: Thank you!

**Review by lindseymoore — APPROVED** [review]
LGTM!

File: docs-release-notes.md (hunk 4 above)
Comment (dacharyc) [reply docs-release-notes.md:12]: FWIW, I'm not manually making these changes. My VS Code is automatically formatting this file on save. We should probably all be applying consistent formatting. We should discuss what linter to use and how to set it up. I'll add a note to the SDK sync document.

**Pull request merged**